server/grpc.go

```diff
@@ -50,46 +50,62 @@ import (
 type Server = grpc.Server
 
 // StartGRPCServerOption is a type for functional style options that can configure the [StartGRPCServer] function.
-type StartGRPCServerOption func(srv *Server, ac *AuthConfig)
+type StartGRPCServerOption func(c *config)
+
+// config contains additional server configurations
+type config struct {
+	grpcOpts   []grpc.ServerOption
+	services   map[*grpc.ServiceDesc]any
```

Comment:
What does "ServiceDesc" mean? Is this an abbreviation for description?

Reply:
Yes, service description